tools/run_danesfield.py

```diff
@@ -224,23 +227,46 @@ def main(args):
 
     p3d_file = config['paths']['p3d_fpath']
 
+    #############################################
+    # Run VisSat pipeline
+    #############################################
+    if args.vissat:
+        aoi_config = config['paths'].get('aoi_config')
+        if aoi_config == None:
+            print("Error: Path to aoi_config file must be provided when using VisSat")
+            exit(1)
+        with open(aoi_config) as f:
+            data = json.load(f)
+
+        vissat_workdir = data['work_dir']
+        cmd_args = ["python3", "/VisSatSatelliteStereo/stereo_pipeline.py", "--config_file", aoi_config]
+        run_step(vissat_workdir, "VisSat", cmd_args)
+        cmd_args = ["python3", "/ply2txt.py", os.path.join(vissat_workdir, 'mvs_results/aggregate_3d/aggregate_3d.ply'), os.path.join(vissat_workdir, 'mvs_results/aggregate_3d/aggregate_3d.txt')]
+        run_step(vissat_workdir, "ply2txt", cmd_args)
+        cmd_args = ["/LAStools/bin/txt2las", "-i", os.path.join(vissat_workdir, 'mvs_results/aggregate_3d/aggregate_3d.txt'), "-parse", "xyz", "-o", p3d_file, "-utm", "17R", "-target_utm", "17R"]
+        run_step(vissat_workdir, "txt2las", cmd_args)
+
     #############################################
     # Find all NTF and corresponding rpc and info
     # tar files
     #############################################
 
     input_paths = []
-    for root, dirs, files in itertools.chain(os.walk(config['paths']['imagery_dir']),
-                                             os.walk(config['paths']['rpc_dir'])):
+    use_rpcs = (config['paths'].get('rpc_dir')!=None)
+    if use_rpcs:
+        iterable = itertools.chain(os.walk(config['paths']['imagery_dir']), os.walk(config['paths']['rpc_dir']))
```

Comment:
I suggest a line break here too. as it was previously
```suggestion
        iterable = itertools.chain(os.walk(config['paths']['imagery_dir']),
                                   os.walk(config['paths']['rpc_dir']))
```